count_hits/reducer.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+prev_page = None
+count_hits = 0
+
+for line in sys.stdin:
+    data = line.strip()
+
+    if len(data) != 1:
+        continue
+    page = data
```

Comment:
This is a bit confusing. Consider renaming `data` to `page` (line 7) instead of assigning